pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -83,6 +102,34 @@ def __exit__(self, exc_type, exc_value, traceback):
         sh("./manage.py lms --settings bok_choy flush --traceback --noinput")
         bokchoy_utils.clear_mongo()
 
+    def verbosity_processes_string(self):
```

Comment:
Something seems off with the way this method is intended to work and the way it appears to be working on the Jenkins build.  It looks like in `scripts/generic-ci-tests.sh` you are passing `-n 1`, which is `DEFAULT_NUM_PROCESSES`.  Looking through this method, it seems like the intent is that result in the "else" version of substring, meaning verbosity should be 2, which isn't happening on Jenkins. I think there is a typecasting issue here where the default values are integers and the command line values are strings.  So anytime num_processes is passed into the command line (even if passing 1), `self.num_processes != 1` will be true.